### PR TITLE
Add extra set up step for nx cloud

### PR DESCRIPTION
**Pull request by rmm101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,10 +200,16 @@ The access token is typically set in the `nx.json` file in your project. You can
 
 ```
 "nxCloud": {
-  "accessToken": "<access-token>"
+  "nxCloudAccessToken": "<access-token>"
```
Comment (gagf90) [inline README.md:203]: why did you change the name field?

Comment (rmm101) [reply README.md:203]: It was different in: https://nx.dev/ci/recipes/security/access-tokens
